ToDoWebApi/TokenOperations/Models/TokenHandler.cs

```diff
@@ -0,0 +1,47 @@
+using System.IdentityModel.Tokens.Jwt;
+using System.Text;
+using Microsoft.IdentityModel.Tokens;
+using ToDoWebApi.Entity;
+
+namespace ToDoWebApi.TokenOperations.Models
+{
+    public class TokenHandler
```

Comment:
Bu class model değil command gibi duruyor, eğer öyleyse yanlış dizinde ve namespace'te.

Reply:
Örnek projede bu şekilde kullanıldığı için bu şekilde bıraktım.